nsm-io/Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache-2.0"
 [dependencies]
 libc = "0.2"
 log = "0.4"
-nix = "0.15"
+nix = "0.20.1"
```

Comment:
Should list it as `"^0.20"`. I also would recommend no listing the patch version, since with SemVer patch signifies a bugfix rather than a feature and it usually a good idea to let fixes in.

Reply:
I just chose the last one supporting rust 1.40. I changed the version now.